1-exercises/B-while-loop/exercise.js

```diff
@@ -1,12 +1,17 @@
 /*
     while loops can be useful when you want to execute some code as long as some condition is true.    
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
+    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-separated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
 function evenNumbers(n) {
     // TODO
+    let i = 0;
+    while (i < n) {
+        i += 2;
+    }
+    console.log(evenNumbers(n));
```

Comment:
Without printing as commas, you can have it like this:
```
function evenNumbers(n) {
  let evenNum = 0;
  let counter = 0;
  while (counter < n) {
    console.log(evenNum);
    evenNum += 2;
    counter++;
  }  
}

let n = 2;
evenNumbers(n);
```

For commas, you can use `array` to keep track of output (i.e. 0,2,4,...) + `join` function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

I leave it as an exercise for you :)